images/common/install-sambacc-common.sh

```diff
@@ -1,18 +1,31 @@
 #!/bin/bash
 
 install_sambacc() {
-    wheeldir="$1"
-    if ! [ -d "${wheeldir}" ]; then
-        echo "no directory: ${wheeldir}"
-        exit 2
+    local distdir="$1"
+    if ! [ -d "${distdir}" ]; then
+        echo "warning: no directory: ${distdir}" >&2
+    else
+        mapfile -d '' artifacts < \
+            <(find "${distdir}" -type f -print0)
     fi
 
-    mapfile -d '' wheels < \
-        <(find "${wheeldir}" -type f -name 'sambacc-*.whl' -print0)
-    mapfile -d '' rpmfiles < \
-        <(find "${wheeldir}" -type f -name '*sambacc-*.noarch.rpm' -print0)
+    local wheels=()
+    local rpmfiles=()
+    local repofiles=()
+    for artifact in "${artifacts[@]}" ; do
+        if [[ ${artifact} =~ sambacc.*\.whl$ ]]; then
+            wheels+=("${artifact}")
+        fi
+        if [[ ${artifact} =~ python.?-sambacc-.*\.noarch\.rpm$ ]]; then
+            rpmfiles+=("${artifact}")
+        fi
+        if [[ ${artifact} =~ sambacc.*\.repo$ ]]; then
+            repofiles+=("${artifact}")
+        fi
+    done
 
 
+    local action=copr
```

Comment:
To better match with other action names used elsewhere in the script:
```suggestion
    local action=install-from-copr-repo
```

Reply:
Done